### PR TITLE
Foundation styling

### PR DESCRIPTION
**Pull request by sublimemarch**

### CODE REVIEW

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
-		Past Events
+		View More Event Albums
 		</a>
 	</h2>
-	
-	<% albums.each do |album| %>
-		<div id=<%= "gallery-anchor-#{album[0]['id']}" %>></div>
-	<% end %>
+
+	<script src='/build/javascripts/events-carousel.js'></script>
```
Comment (sowiecki) [inline app/views/events/index.html.erb]: This file no longer exists, please remove this script tag.

Comment (sublimemarch) [inline app/views/events/index.html.erb]: Resolved. :)

Comment (thewildpendulum) [inline app/views/events/index.html.erb]: @stephmarx Looks like the wrong line got removed.

**Comment by thewildpendulum** [comment]
Thanks, @stephmarx :+1:

**Pull request merged**